orator/query/builder.py

```diff
@@ -1064,7 +1064,7 @@ def _run_select(self):
             self.to_sql(), self.get_bindings(), not self._use_write_connection
         )
 
-    def paginate(self, per_page=15, current_page=None, columns=None):
+    def paginate(self, per_page=15, current_page=None, column_count=None, columns=None):
```

Comment:
Since `column_count` has a default value, can you move this as a last param ? That way, existing code won't break if caller passes only 3 arguments, thinking 3rd one is `column`